### ISSUE TITLE
Video call option is available on mobile app but not on desktop App

### ISSUE THREAD

**Issue by Nirzak** [telegramdesktop/tdesktop#16147]
### Expected behaviour
Video call option should be seen beside audio call option just like the mobile app.

### Actual behaviour
But It is not available.

### Configuration
**Operating system:** Windows 10

**Version of Telegram Desktop:** 2.7.1

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:

**Comment by john-preston** [comment]
@Nirzak You can turn the video on after the audio call is established.

**Comment by Nirzak** [comment]
> @Nirzak You can turn the video on after the audio call is established.

Yeah but a separate option can be helpful to initiate it directly

**Comment by tarcisioe** [comment]
What makes this quite a bit more annoying is that if you had a previous video call and click on the message, it starts a video call directly, which means the feature does in fact exist.

**Comment by julianxhokaxhiu** [comment]
I agree, this is very annoying. Could this at least be added as a "right click" popup con the call button? As the behavior exists, it's all about adding a shortcut in the UI.

**Comment by marcorivm** [comment]
+1

**Comment by jeffdcamp** [comment]
Biggest issue I have is when I'm wanting video call someone, especially a mobile device.... I have to then tell them to switch to speaker and video (both the caller and the receiver have to go through the process of turn on video AND turning on the speaker (if calling a mobile device)).

This has causes me to only use my phone to make video calls :(  (especially for video calling elderly people)

I think it would be good to be consistent on all platforms: mobile, web, and desktop (both mobile and web allow you to start a video call... only desktop does not allow you to do this)

**Comment by Aokromes** [comment]
well now you have

![Image](https://github.com/user-attachments/assets/c982ff6f-0257-4a38-9ae6-5a93b9618acd)
